setup.py

```diff
@@ -63,7 +63,7 @@
             "statsmodels>=0.12.2",
             "psutil==5.8.0",
             "dataclasses",
-            "transformers[torch]==4.18",
+            "transformers[torch]",
```

Comment:
This change is OK as it would test the latest version of transformers in the build action, so that we would know when a future release of transformers breaks our test.